packages/kbn-search-response-warnings/src/extract_warnings.ts

```diff
@@ -24,24 +23,14 @@ export function extractWarnings(
   const warnings: SearchResponseWarning[] = [];
 
   const isPartial = rawResponse._clusters
-    ? Object.values(
-        (
-          rawResponse._clusters as estypes.ClusterStatistics & {
-            details: Record<string, ClusterDetails>;
-          }
-        ).details
-      ).some((clusterDetails) => clusterDetails.status !== 'successful')
```

Comment:
It's possible to have clusters where the status is `'running'`, which would have been considered partial before this PR, but won't after this PR. Is that the intended behavior or should we add `|| rawResponse._clusters.running > 0` here?

Reply:
> Is that the intended behavior

No its not. Thank you for pointing this out. I have added your suggestion of enhancing the logic with a check for running.  https://github.com/elastic/kibana/pull/171369/commits/064d8e78ce22cc6d9fdb8e0ac469ef1064ba1ae6